app/policies/claims/claim_policy.rb

```diff
@@ -1,6 +1,10 @@
 class Claims::ClaimPolicy < Claims::ApplicationPolicy
+  def create?
+    Claims::ClaimWindow.current.present?
+  end
+
   def edit?
-    !record.submitted?
+    create? && !record.submitted?
```

Comment:
This needs to check if the claim's window it was submitted within is current or not.